addon/components/base-chart-component.js

```diff
@@ -288,6 +289,29 @@ export default Component.extend({
         this.get('chart').render();
     },
 
+    _getBaseChart(type) {
```

Comment:
Could add JSDoc here saying the passed chart will be one of the defined types in `ember-data-visualizations/utils/chart-types`